src/Dfc.CourseDirectory.Core/IFeatureFlagProvider.cs

```diff
@@ -4,6 +4,6 @@ namespace Dfc.CourseDirectory.Core
 {
     public interface IFeatureFlagProvider
     {
-        ISet<string> GetFeatureFlags();
+        IReadOnlyCollection<string> GetFeatureFlags();
```

Comment:
Moved to `IReadOnlyCollection` as discussed with JG - once we're at .NET 5, we can use `IReadOnlySet` instead.